src/app_service/service/accounts/service.nim

```diff
@@ -293,7 +293,8 @@ QtObject:
           "text": ""
         },
       "profile-pictures-show-to": settings.PROFILE_PICTURES_SHOW_TO_EVERYONE,
-      "profile-pictures-visibility": settings.PROFILE_PICTURES_VISIBILITY_EVERYONE
+      "profile-pictures-visibility": settings.PROFILE_PICTURES_VISIBILITY_EVERYONE,
+      "url-unfurling-mode": 1,
```

Comment:
"Magical constant" code smell ;)

Reply:
I wonder why I even did this 😄 
fixed 👌